### ISSUE TITLE
Strange behaviour of the websocket egress bitrate when microphone is reduced

### ISSUE THREAD

**Issue by samhumeau** [livekit/egress#145]
## The regular situation

When publishing audio with the `livekit-cli` `publish` capability and starting a websocket egress, I measure a bitrate of 192kB/s on the receiving side. This is exactly what we would expect as
```
48 kHz sample rate * 2 bytes per sample * 2 channels = 192kB/s
```
The rate oscillate around this value.

https://user-images.githubusercontent.com/4238961/197003743-51b73d69-892f-4f3a-a429-89d5bea2bc3b.mov

## The weird situation

Now when the audio is published via the js client, on chrome on mac, if I reduce the microphone the bitrate suddenly decreases to around 20kB/s. The following video shows the behavior:

https://user-images.githubusercontent.com/4238961/197007617-b9faedfc-6326-4a79-96be-56d7847cb743.mp4

## Expected situation
The bitrate is constant, or some mute event is sent.

## Config:
```
Macbook pro M1
Chrome 106.0
livekit client js 1.1.7
livekit server: 1.2.1
livekit egress: 1.4.0
```

**Comment by frostbyte73** [comment]
Hey @samhumeau, this is expected - opus is designed to save bandwidth when there is less data being sent. If you have dtx enabled, this can be as little as one frame every 400ms. If you want a mute event, you can mute the track programmatically instead of by dropping the input volume

**Comment by frostbyte73** [comment]
Sorry, I misunderstood - the websocket is receiving raw PCM, not opus. I just merged #146, which should result in a constant bitrate
